tests/ignite/metrics/test_ndcg.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+import pytest
+import torch
+from sklearn.metrics import ndcg_score
+
+from ignite.exceptions import NotComputableError
+from ignite.metrics.recsys.ndcg import NDCG
+
+
+@pytest.mark.parametrize(
+    "true_score, pred_score",
+    [
+        (torch.tensor([[1.0, 2.0, 3.0, 4.0, 5.0]]), torch.tensor([[0.1, 0.2, 0.3, 0.4, 0.5]])),
+        (torch.tensor([[2.9, 5.6, 3.8, 7.9, 6.2]]), torch.tensor([[3.7, 4.8, 3.9, 4.3, 4.9]])),
+        (
+            torch.tensor([[1.0, 2.0, 3.0, 4.0, 5.0], [1.2, 4.5, 8.9, 5.6, 7.2], [2.9, 5.6, 3.8, 7.9, 6.2]]),
+            torch.tensor([[0.1, 0.2, 0.3, 0.4, 0.5], [3.7, 4.8, 3.9, 4.3, 4.9], [3.7, 4.8, 3.9, 4.3, 4.9]]),
+        ),
+    ],
+)
+@pytest.mark.parametrize("k", [None, 2, 3])
```

Comment:
Let's try to think how can we test for other options like : 
- exponential
- log_base

For exponential, maybe we still can use sklearn and rescale inputs ?

Reply:
Thats right

Reply:
For log_base I can rewrite the sklearn function?

Reply:
Yes, we can try that

Reply:
Sure, will have a look